Sources/BottomSheet/Detent.swift

```diff
@@ -21,6 +22,8 @@ public enum Detent {
             return 0.50
         case .small:
             return 0.20
+        case .hidden:
+            return 0.0
         case .fraction(let value):
             return value
```

Comment:
When we are working with Detents, maybe we should check if value is not negative? There is no sense with that value for .fraction